docs/csharp/programming-guide/concepts/linq/xelement-class-overview.md

```diff
@@ -12,9 +12,9 @@ The <xref:System.Xml.Linq.XElement> class is one of the fundamental classes in [
 ### Constructing XML Trees  
```

Comment:
This is really completely unrelated to your PR, but the "XElement Functionality" head on line 9 is useless, and as a result its child heads don't appear in the TOC. I'd consider deleting it, leaving the following sentence as the next paragraph of the introduction, and promoting each level-3 head one level.